metadata.yaml

```diff
@@ -38,3 +39,7 @@ resources:
     type: file
     filename: calico-upgrade.tar.gz
     description: 'calico-upgrade tool for arm64'
+  calico-node-image:
```

Comment:
Remind me, this resource was added as a workaround for the issue where `ctr image pull` does not properly use the *_proxy configs from the containerd subordinate, correct?

The resource will need to be added to our build automation:
* https://github.com/charmed-kubernetes/layer-calico/blob/master/build-calico-resource.sh
* https://github.com/charmed-kubernetes/jenkins/blob/master/jobs/build-charms/resource-spec.yaml

Reply:
That's correct @Cynerva, so we'll have to add it as you say.